README.md

```diff
@@ -84,7 +84,10 @@ wis2node data setup --topic-hierarchy foo.bar.baz
 wis2node data info --topic-hierarchy foo.bar.baz
 
 # process incoming data (manually/no PubSub)
-wis2node data process /path/to/file.csv
+wis2node data ingest /path/to/file.csv
```

Comment:
Should this be clearer that if the topic hierarchy is not specified, the path to the file should contains the directory path? It could be left out until we do a bigger sweep on documentation?

Reply:
Good point @webb-ben.  Updated.